Formula/postgresql@14.rb

```diff
@@ -104,41 +112,94 @@ def post_install
   end
 
   def postgresql_datadir
-    var/"postgres"
+    if old_postgres_data_dir.exist?
+      old_postgres_data_dir
+    else
+      var/name
+    end
   end
 
   def postgresql_log_path
-    var/"log/postgres.log"
+    var/"log/#{name}.log"
   end
 
   def pg_version_exists?
     (postgresql_datadir/"PG_VERSION").exist?
   end
 
+  def old_postgres_data_dir
+    var/"postgres"
+  end
+
+  def postgresql_formula_present?
+    Formula["postgresql"].any_version_installed?
```

Comment:
This is throwing a warning in `brew audit`: https://github.com/Homebrew/homebrew-core/runs/7856132015?check_suite_focus=true#step:6:30

Not sure if it's worth doing something about.

Reply:
Yeh, should fix, thanks.